Makefile

```diff
@@ -159,6 +159,9 @@ clean: #HELP Clean up build artifacts
 
 #SECTION Development
 
+.PHONY: sanity-check
+sanity-check: lint vet #HELP Lint and vet
```

Comment:
Is it worth having a new target for this? Can we `make lint vet` in CI?

Reply:
fair point - I just thought it be easier to update the Makefile rather than look for the right place in GHA (but that could just as much be an argument against). I'll remove.

Reply:
Fixed